### PR TITLE
Fix the issue about placeholder was not cleared in textarea.

### PR DESCRIPTION
**Pull request by wuhengzhi**
This patch is to fix the issue about placeholder was not cleared in
textarea when enter first character.
Invoke the construction for AutofillAgent(), it will set auto fill
client which will be used when the text was changed in textarea.

BUG=XWALK-3334

### CODE REVIEW

**Comment by seme1** [comment]
How can we apply the fix to the laest stable/beta version of crosswalk-cordova ? Is it possible to include the fix in the latest beta/stable versions of crosswalk-cordova ?
My app is almost ready to publish on the Play Store and I'm just waiting for this issue to be fixed.

**Comment by halton** [comment]
> How can we apply the fix to the laest stable/beta version of crosswalk-cordova ? Is it possible to include the fix in the latest beta/stable versions of crosswalk-cordova ?
My app is almost ready to publish on the Play Store and I'm just waiting for this issue to be fixed.

@seme1 , if a fix is important, we will promote the change (normally cherry-pick and new a PR) into beta/stable. @rakuco know it better.

File: runtime/renderer/xwalk_content_renderer_client.cc [hunk 1]
```diff
@@ -156,6 +158,11 @@ void XWalkContentRendererClient::RenderViewCreated(
 #elif defined(OS_TIZEN)
   XWalkRenderViewExtTizen::RenderViewCreated(render_view);
 #endif
+  // TODO(sgurun) do not create a password autofill agent (change
```
Comment (halton) [inline runtime/renderer/xwalk_content_renderer_client.cc]: saurun is not you, correct?

I'd suggest to add comment to say the code is copied from `src/android_webview/renderer/aw_content_renderer_client.cc`, as long as the TODO.

**Comment by wuhengzhi** [comment]
@halton I have added the comment. Thanks.

**Comment by halton** [comment]
LGTM

File: runtime/renderer/xwalk_content_renderer_client.cc [hunk 2]
```diff
@@ -156,6 +158,13 @@ void XWalkContentRendererClient::RenderViewCreated(
 #elif defined(OS_TIZEN)
   XWalkRenderViewExtTizen::RenderViewCreated(render_view);
 #endif
+  // The following code was copied from
+  // android_webview/renderer/aw_content_renderer_client.cc
+  // TODO(sgurun) do not create a password autofill agent (change
+  // autofill agent to store a weakptr).
+  autofill::PasswordAutofillAgent* password_autofill_agent =
+      new autofill::PasswordAutofillAgent(render_view);
+  new autofill::AutofillAgent(render_view, password_autofill_agent, NULL);
```
Comment (pozdnyakov) [inline runtime/renderer/xwalk_content_renderer_client.cc]: nit: nullptr

**Comment by pozdnyakov** [comment]
lgtm with a nit

**Comment by wuhengzhi** [comment]
@pozdnyakov Done, thanks.

**Comment by rakuco** [comment]
This patch has broken the build, most likely because the Chromium code changed after the M41 rebase.

**Comment by rakuco** [comment]
> This patch has broken the build, most likely because the Chromium code changed after the M41 rebase.

I've sent a proposed fix in #2848.

**Pull request merged**